build.gradle

```diff
@@ -168,6 +168,8 @@ subprojects {
         googleCloudGaVersion = '1.65.0'
         log4j2Version = '2.11.1'
         signalfxVersion = '0.0.48'
+        springBoot2Version = '2.1.5.RELEASE'
+        springBootTest2Version = '2.1.5.RELEASE'
```

Comment:
Nit: this isn't needed since it's the same as `springBoot2Version`

Reply:
done.